README.md

```diff
@@ -2681,6 +2681,7 @@ _You can enable the following settings in Xcode by running [this script](resourc
   * `// MARK: Lifecycle` for `init` and `deinit` methods.
   * `// MARK: Open` for `open` properties and methods.
   * `// MARK: Public` for `public` properties and methods.
+  * `// MARK: Package` for `package` properties and methods.
```

Comment:
This ordering makes sense to me, based on this description from the proposal.
<img width="1073" alt="Screenshot 2023-07-06 at 1 41 43 PM" src="https://github.com/airbnb/swift/assets/1791049/b5d90d23-40dd-4956-8d55-c5c625f9cb2c">